recipe2023/index.css

```
@@ -0,0 +1,50 @@
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 1080px;
+  padding-top: 5%;
+}
+
+.centered-text {
+  text-align: center;
+}
+.main-title {
+  padding: 500px;
+}
+.highlight-title {
+  padding-bottom: 10%;
+}
+
+.column {
+  /* //splits half and half: allows us to create two rows in this case */
+  flex: 50%;
+}
+
+/* Clear floats after the columns */
+.row {
+  display: flex;
+}
+
+.removebullets {
+  margin-left: 100px;
+  margin-right: 100px;
+}
+.removebullets2 {
+  list-style: none;
+  margin-left: 100px;
+  margin-right: 100px;
+}
+
+.paragraph {
+  margin-left: 150px;
+  margin-right: 150px;
+}
+.paragraph2 {
+  margin-left: 150px;
+  margin-right: 150px;
+}
+
```

```suggestion
.paragraph {
  margin-left: 150px;
  margin-right: 150px;
}
```

If you have two classes doing the exact same thing, no need to duplicate it. You can just reuse the class in your HTML as many times as you want